src/Workspaces/CSharp/Portable/CodeGeneration/CSharpCodeGenerationService.cs

```diff
@@ -637,37 +626,26 @@ public override SyntaxNode CreateFieldDeclaration(IFieldSymbol field, CodeGenera
             return NamespaceGenerator.GenerateNamespaceDeclaration(this, @namespace, options, cancellationToken);
         }
 
-        private static TDeclarationNode UpdateDeclarationModifiers<TDeclarationNode>(TDeclarationNode declaration, Func<SyntaxTokenList, SyntaxTokenList> computeNewModifiersList, CodeGenerationOptions options, CancellationToken cancellationToken)
-        {
-            switch (declaration)
+        private static TDeclarationNode UpdateDeclarationModifiers<TDeclarationNode>(TDeclarationNode declaration, Func<SyntaxTokenList, SyntaxTokenList> computeNewModifiersList, CodeGenerationOptions options)
+            => declaration switch
             {
-                case BaseTypeDeclarationSyntax typeDeclaration:
-                    // Handle type declarations.
```

Comment:
seems like these comments are not the most important, just wondering whether they were deleted manually or omitted by the fixer?

Reply:
i omitted them because they were pointless :D